production/ksonnet/loki/multi-zone.libsonnet

```diff
@@ -49,16 +56,65 @@
   ingester_zone_b_args:: {},
   ingester_zone_c_args:: {},
 
-
-  // For migration purposes we need to be able to configure a zone for single ingester statefulset deployments.
   ingester_container+:: if !$._config.multi_zone_default_ingester_zone then {} else
     container.withArgs($.util.mapToFlags($.ingester_args {
       'ingester.availability-zone': 'zone-default',
     })),
 
-  newIngesterZoneContainer(zone, zone_args)::
-    local zone_name = 'zone-%s' % zone;
+  // The ingesters should persist TSDB blocks and WAL on a persistent
+  // volume in order to be crash resilient.
+  local ingester_data_pvc =
```

Comment:
Looks like we are repeating a bunch of code from `ingester.libsonnet`. I wonder if we should reuse some of the things we define on that file here.